### PR TITLE
21464 - Merge EFT into main branch

### PR DESCRIPTION
**Pull request by seeker25**
*Issue #:*
https://github.com/bcgov/entity/issues/21464 

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

**Comment by seeker25** [comment]
CD's probably will fail for GCP, ignore those.. we can fix those in another ticket

**Comment by seeker25** [comment]
Will fix notebook reports now

**Comment by seeker25** [comment]
Fixing 12 codecov issues

**Comment by ochiu** [comment]
Thanks for powering through this.
Looks good to me, will check back as I know you are taking a look at some of the CI issues.

**Review by ochiu — APPROVED** [review]

File: bcol-api/requirements.txt [hunk 1]
```diff
@@ -1,51 +1,49 @@
 Flask-Moment==1.0.5
```
Comment (seeker25) [inline bcol-api/requirements.txt:1]: Will remove

File: pay-api/gunicorn_config.py [hunk 2]
```diff
@@ -18,11 +18,11 @@
...
 
 # https://docs.gunicorn.org/en/stable/settings.html#workers
-workers = int(os.environ.get('GUNICORN_PROCESSES', '3'))  # gunicorn default - 1
+workers = int(os.environ.get('GUNICORN_PROCESSES', '1'))  # gunicorn default - 1
 worker_class = os.environ.get('GUNICORN_WORKER_CLASS', 'sync')  # gunicorn default - sync
 worker_connections = int(os.environ.get('GUNICORN_WORKER_CONNECIONS', '1000'))  # gunicorn default - 1000
-threads = int(os.environ.get('GUNICORN_THREADS', '1'))  # gunicorn default - 1
-timeout = int(os.environ.get('GUNICORN_TIMEOUT', '100'))  # gunicorn default - 30
+threads = int(os.environ.get('GUNICORN_THREADS', '8'))  # gunicorn default - 1
+timeout = int(os.environ.get('GUNICORN_TIMEOUT', '0'))  # gunicorn default - 30
```
Comment (seeker25) [inline pay-api/gunicorn_config.py:25]: I think I'll have to revert this

**Review by Jxio — APPROVED** [review]

**Pull request merged**